### DISCUSSION TITLE
Awesome component

### DISCUSSION THREAD

**Discussion by eonist** [nschloe/stackoverflow-card#9] [Q&A]
Any chance it will go down in the future? How is hosting cost etc?

**Comment by nschloe** [comment]
It's really hard to answer this. Any chance that it will go down? Sure! Probably when I lose interest in this.

**Reply by eonist** [reply]
Does it cost money to host? Does Vercel have free hosting for public code like this?

**Reply by nschloe** [reply]
It doesn't now, it might in the future.

**Reply by nschloe** [reply]
Out of interest, why did you want to know?

**Reply by Laaouatni** [reply]
The service is down unfortunately. Can I host myself this thing? (I really need it for my github readme) and how to do it


------


the images don't show anymore.

when I investigate even more, the api says: `{"error":"Failed fetching data"}`

![image](https://github.com/nschloe/stackoverflow-card/assets/87947051/dac1dcc9-eb4d-4405-b42d-b046a422b589)

------

this happens not only on my readme, but on every one since even the official docs don't show anymore the image.


![image](https://github.com/nschloe/stackoverflow-card/assets/87947051/b0a190f6-ee47-44b9-b8ef-6160403a39f2)

**Comment by eonist** [comment]
Pimped my GitHub today. https://github.com/eonist And your 1 of 3 suppliers of markdown stackoverflow dash. And the other two are not as good as yours. Also I don't know how easy it is to host stuff on Vercel these days. If they let stuff live on even with high loads of traffic etc.

**Comment by eonist** [comment]
Are many using it?

**Comment by eonist** [comment]
@Laaouatni It comes and goes. Mostly works tho.
